Mods/SML/Source/SML/Private/Patching/BlueprintHookManager.cpp

```diff
@@ -169,16 +227,48 @@ void UBlueprintHookManager::HookBlueprintFunction(UFunction* Function, const TFu
 	}
 #endif
 
+	//Make sure to add outer UClass to root set to avoid it being Garbage Collected
+	//Because otherwise after GC script byte code will be reloaded, without our hooks applied
+	UClass* OuterUClass = Function->GetTypedOuter<UClass>();
+	check(OuterUClass);
+	HookedClasses.AddUnique(OuterUClass);
```

Comment:
I would keep this part above the `#if UE_BLUEPRINT_EVENTGRAPH_FASTCALLS` block

Reply:
Good catch, fixed.

Reply:
Ack